cognite/client/_api/files.py

```diff
@@ -1283,7 +1281,7 @@ def download_bytes(
 
     def _download_file(self, download_link: str) -> bytes:
         res = self._http_client_with_retry.request(
-            "GET", download_link, accept="*/*", timeout=self._config.file_transfer_timeout
+            "GET", download_link, headers={"accept": "*/*"}, timeout=self._config.file_transfer_timeout
```

Comment:
These are unreleated changes?

Reply:
Yup, the accept parameter was misleading. It should have been passed in headers.